specification/metrics/pricingquantity.md

```diff
@@ -0,0 +1,31 @@
+# Pricing Quantity
+
+The Pricing Quantity represents the volume of a given resource and/or service used or purchased based on the [Pricing Measure](#pricingmeasure). Distinct from [Usage Quantity](#usagequantity) (complementary to [Usage Unit](#usageunit)), it focuses on pricing and cost, not resource and service consumption.
+
+PricingQuantity MUST be present in the billing data. This column MUST be a numeric value of type Decimal. The value MAY be negative in cases where [ChargeType](#chargetype) is "Adjustment". This column MUST NOT contain null values where unit prices are not null. When unit prices are not null, multiplying PricingQuantity by a unit price MUST produce a result equal to the corresponding cost metric.
```

Comment:
Minor suggestion (feel free to resolve this comment) about:

>  multiplying PricingQuantity by a unit price MUST produce a result equal to the corresponding cost metric.

The FOCUS names of unit price and cost metrics could be also explicitly mentioned to ease user understanding.

Reply:
The math is applicable to all unit price and cost metrics. In version 1.0, we only have ListUnitPrice, and in the future, we'll likely introduce NegotiatedUnitPrice (name subject to change). The idea is to avoid changing this spec when/if we add additional columns or rename existing ones.